ROS/controls/src/controls_constants.py

```diff
@@ -0,0 +1,26 @@
+
+# The maximum number of ROS messages that
+# are buffered by publishers in the controls package
+CONTROLS_PKG_PUBLISHER_QUEUE_SIZE = 10
+
+# The maximum number of ROS messages that
+# are buffered by subscribers in the controls package
+CONTROLS_PKG_SUBSCRIBER_QUEUE_SIZE = 10
+
+# The topic name through which the desired orientation
+# of the motors is communicated
+CONTROLS_PKG_DESIRED_ORIENTATION_TOPIC = 'desired_orientation'
+
+# The topic name through which the current orientation
+# of the motors is communicated
+CONTROLS_PKG_CURRENT_ORIENTATION_TOPIC = 'current_orientation'
+
+# The default rate at which the Moteus motors are queried
+# for their current state (defaults to 60 times a second)
+CONTROLS_PKG_DEFAULT_HW_REFRESH_RATE = 60
+
+# A mapping of the Moteus motor IDs with CAN bus lanes
+CONTROLS_PKG_BUS_CONFIGURATION = {1 : 0, 2 : 0}
+
+# The name of the node that runs on the RaspberryPi
+CONTROLS_PKG_RASPBERRYPI_NODE_NAME = 'controls_raspberrypi_node'
```

Comment:
I'd reverse the order of this naming for it to be quickly evident on which system it runs; something along the lines of

rasp_pi_control_node

Reply:
Do you mean for the name itself or for all of the constants? I think you mean the former but just want to clarify

Reply:
Both. Overall, this is admittedly nit picky, but it may help improve readability. Ultimately leaving it to your discretion -just something to consider